packages/cashier/src/types/shared/account.types.ts

```diff
@@ -47,4 +47,5 @@ export type TAccountsList = {
     icon?: string;
     idx: string | number;
     is_dark_mode_on: boolean;
+    title: string;
```

Comment:
The `TAccountsList` is used in some components and adding (required) property will cause error in those components. I think we can make It optional or check this type in the components that are using this type to check for errors.